source/process/sg_general-guidelines.rst

```diff
@@ -0,0 +1,92 @@
+.. _general-guidelines:
+
+General Guidelines
+==================
+
+The Mattermost documentation must be of high quality. It must be accurate and clear, and be presented with a style and tone that is appropriate for technical content. People who use Mattermost rely on the documentation to get their jobs done. We don't want to see an installation of Mattermost delayed because the documentation has an error or is difficult to understand.
+
+Style Guide Applicability
+-------------------------
+
+The style guide applies to all end-user, developer, and administrator documentation that appears on the Mattermost documentation site at https://docs.mattermost.com.
+
+Apply the style guide in the following situations:
+
+- when you create a new document
+- when you convert a document from Markdown to reStructuredText
+- when you revise a document
+
+When you revise a document, apply the style guide rules to the part that you changed. If you have time to update the rest of the document, then do so. If updating the style for an entire document impacts a release, or if the scope is significantly greater than you originally anticipated, then do what you can and what makes sense, and then create an `issue in GitHub <https://github.com/mattermost/docs/issues>`_. Give the issue an appropriate title, such as *File XXX converted to .rst, but needs updating for style guide*.
```

Comment:
"If updating the style for an entire document impacts a release" > I'm not sure what this means